tailnet/proto/tailnet.proto

```diff
@@ -108,8 +108,7 @@ message IPFields {
 		PUBLIC = 0;
 		PRIVATE = 1;
 		LINK_LOCAL = 2;
-		UNIQUE_LOCAL = 3;
-		LOOPBACK = 4;
+		LOOPBACK = 3;
```

Comment:
As per Slack discussion, `UNIQUE_LOCAL` was removed in favor of `{ v=4, class=PRIVATE }` meaning "ipv4 private range" and `{ v=6, class=PRIVATE }` meaning unique local.